### ISSUE TITLE
shared atlas db connection shows only one database

### ISSUE THREAD

**Issue by tothradoslav** [mingo-app/mingo#431]
Hi guys! I have a question. I'm doing a connection to a shared atlas db, the connection is a type SRV URI, when I put the URI complete in another clients they are connecting and listing all the databases they found, In mingo the connection is success but only shows the admin database and nothing else.

Source: discord @estebanx64

**Comment by Icaruk** [comment]
Same issue here. I have 2 URIs with that problem.

I fixed the first one removing `/admin` as database.

```
...mongodb.net/admin?authSource=admin&replicaSet=...
```
> 1 database found: admin

```
...mongodb.net/?authSource=admin&replicaSet=...
```
> All databases found.

But that fix didn't worked on my second URI, only "admin" database was found.

**Comment by Icaruk** [comment]
Any news on this? :(

**Comment by tothradoslav** [comment]
Hi, sorry for the late reaction here... :(

We found the issue, ready for next release. Please keep the issue open until release.

**Comment by tothradoslav** [comment]
A new Mingo release candidate is available for download (v1.10.0-rc2). This issue has been fixed in this release. Please check it out and let us know if it works as expected. Thank you!

https://github.com/mingo-app/mingo/releases/tag/v1.10.0-rc2
